ztp/ztp-policy-generator/README.md

```diff
@@ -0,0 +1,318 @@
+# Policy Generator (PolicyGen)
+
+The policy generator (PolicyGen) is a kustomize plugin used to facilitate creating ACM policies from a predefined Custom Resources (CRs). There are 3 main items (Policy Categorization, Source CR policy and PolicyGenTemplate) that PolicyGen rely on to generate the ACM policies and its placement binding and placement rule.
+
+1 - Policy Categorization:
+
+ Any generated policy needs to be placed under one of the following categories; 
```

Comment:
It might be helpful to explain that common is a single category but the user can create multiple groups and sites.

Reply:
okay, will add more explain for the categories